src/main/resources/resources/GroupNameResources_en.properties

```diff
@@ -23,7 +23,9 @@ label.name.AlumniGroup = Alumni{0}
 label.name.CampusEmployeeGroup = Employees of campus {0}
 label.name.CandidateGroup = Candidates
 label.name.CerimonyInquiryGroup = Inquired persons of {0}
-label.name.CoordinatorGroup = Coordination{0}{1}
+label.name.CoordinatorGroup= {0}Coordination{1}{2}
```

Comment:
is this correct ?

Reply:
Yes, because {0} matches the responsibleQualifier parameter.
Which means, in this case (EN), we get Responsible Coordination/Assistant Coordination.

{1} and {2} match the degreeType and degree parameters (previously {0} and {1}).